### PR TITLE
[W4.11e][T09-3]Benedict

### PR DESCRIPTION
**Pull request by benedictcss**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +62,8 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsNull(Object... objects) { assertTrue(Utils.isAnyNull(objects)); }
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:66]: As mentioned in the tutorial, we usually don't recommend single line method.

Comment (eugenepeh) [reply test/java/seedu/addressbook/common/UtilsTest.java:66]: Also, no need to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -10,6 +10,25 @@
...
+
+        // multiple objects with null
+        assertIsNull(null, "abc");
+        assertIsNull("abc", null, "ab");
+        assertIsNull("abc", "ab", "a", null);
+
+        // multiple objects without null
+        assertNotNull("a", "ABC");
+        assertNotNull("abc", "ab", "a");
+        assertNotNull("abc", "ab", "a", "ABC");
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:30]: Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.

Comment (eugenepeh) [reply test/java/seedu/addressbook/common/UtilsTest.java:30]: Good job on adding both positive and negative test cases, as well as empty and null scenario though.

**Review by eugenepeh — COMMENTED** [review]
Good job on completing the activity.
Some comments added. Please close the PR after reading the comments.
